pkg/storageincentives/redistribution/redistribution.go

```diff
@@ -209,3 +211,15 @@ func (c *contract) callTx(ctx context.Context, callData []byte) ([]byte, error)
 	}
 	return result, nil
 }
+
+func (c *contract) setFee(ctx context.Context, txHash common.Hash) {
+	fee, err := c.txService.TransactionFee(ctx, txHash)
```

Comment:
does this method needs to be guarded with mutex when value of `accumulativeFee` is set?

Reply:
https://github.com/ethersphere/bee/blob/a670af9fb998e88c57dfac43ad7e6b6f01ddfd78/pkg/storageincentives/nodestatus.go#L105